Packs/NetscoutAED/Integrations/NetscoutAED/NetscoutAED.yml

```diff
@@ -0,0 +1,1286 @@
+    deprecated: false
+    description: Removes one or more countries from the blacklist for a specific protection
+      group or for all protection groups.
+    execution: false
+    name: na-aed-inbound-blacklisted-countries-remove
+  - arguments:
+    - default: false
+      description: A comma-separated list of IPv4 host addresses or CIDRs.
+      isArray: false
+      name: host_address
+      required: false
+      secret: false
+    - default: false
+      description: 'Search strings, separated by “+” to filter the results. (example:
+        "AZ+BS").'
+      isArray: false
+      name: query
+      required: false
+      secret: false
+    - default: false
+      description: The page of the results to return.
+      isArray: false
+      name: page
+      required: false
+      secret: false
+    - default: false
+      description: Maximal number of results to retrieve. Also sets the size of the
+        returned page.
+      isArray: false
+      name: limit
+      required: false
+      secret: false
+    deprecated: false
+    description: Gets the outbound blacklisted hosts. By default, 10 blacklisted hosts
+      are returned.
+    execution: false
+    name: na-aed-outbound-blacklisted-hosts-list
+    outputs:
+    - contextPath: NetscoutAED.OutboundBlacklistHost.annotation
+      description: A description of the host.
+      type: String
+    - contextPath: NetscoutAED.OutboundBlacklistHost.host_address
+      description: IPv4 host address or CIDRs.
+      type: String
+    - contextPath: NetscoutAED.OutboundBlacklistHost.update_time
+      description: The time the host was last updated/set
+      type: Date
+  - arguments:
+    - default: false
+      description: A single IPv4 host address or CIDR or a comma-separated list of
+        IPv4 host addresses or CIDRs.
+      isArray: true
+      name: host_address
+      required: true
+      secret: false
+    - default: false
+      description: A single description that applies to all of the specified hosts
+        or a comma-separated list of descriptions, each of which applies to a specific
+        host.
+      isArray: false
+      name: annotation
+      required: false
+      secret: false
+    deprecated: false
+    description: Adds one or more hosts to the outbound blacklist.
+    execution: false
+    name: na-aed-outbound-blacklisted-hosts-add
+    outputs:
+    - contextPath: NetscoutAED.OutboundBlacklistHost.annotation
+      description: A description of the host.
+      type: String
+    - contextPath: NetscoutAED.OutboundBlacklistHost.host_address
+      description: IPv4 host address or CIDRs.
+      type: String
+    - contextPath: NetscoutAED.OutboundBlacklistHost.update_time
+      description: The time the host was last updated/set.
+      type: Date
+  - arguments:
+    - default: false
+      description: A single IPv4 host address or CIDR or a comma-separated list of
+        IPv4 host addresses or CIDRs.
+      isArray: false
```

Comment:
If this can be comma separated values argument, then 'isArray' should be set to 'true'.
Please verify that's true for all comma-separated list arguments.

Reply:
Done